models/time_cache.cr

```diff
@@ -0,0 +1,26 @@
+struct TimeCache(K, V)
```

Comment:
It could belong there, in the standard library, or a different project, but I don't know if this is general enough.